pkgs/by-name/tt/ttaenc/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "ttaenc";
+  version = "3.4.1";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/tta/${finalAttrs.name}-${finalAttrs.version}-src.tgz";
+    sha256 = "sha256-ssnIsBWsxYZPCCoBV/LgnFEX0URTIctheOkltEi+PcY=";
+  };
+
+  makeFlags = [ "INSDIR=$(out)/bin" ];
+
+  preBuild = ''
+    # From the Makefile, with `-msse` removed, since we have those on by x86_64 by default.
+    makeFlagsArray+=(CFLAGS="-Wall -O3 -fomit-frame-pointer -funroll-loops -fforce-addr -falign-functions=4")
```

Comment:
just out of curiosity, is there a reason you used this here instead of just adding to the `makeFlags` attribute? does it operate semantically differently?

Reply:
I think this has something to do with my CFLAGS having spaces inside. I wasn't able to figure out how to do that with `makeFlags`, and saw some other nixpkgs derivation doing this instead, so I copied